test-requirements.txt

```diff
@@ -5,3 +5,4 @@ pytest-cov
 pytest-helpers-namespace
 pytest-mock
 wheel
+gilt-python
```

Comment:
Great cleanups.  Could you alphabetize the new dep?  Other than that looks good!